data/dump/count_info.data.R

```diff
@@ -1,5 +1,7 @@
 N <- 259
 P <- 18
+str <- 
+c(1, 1, 1, 1, 1, 10, 1, 10, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1)
 end <- 
 c(259, 259, 259, 259, 259, 250, 259, 250, 259, 259, 259, 259, 259, 259,
```

Comment:
Continuing from my note in `data.r`, the first named for group 6 is at t==10, as seen in `str`. Now, the end is at 250, should it be at 259 across each group? Otherwise, it seems we are subsetting from t==10 to t==250 and chopping off real values > 250